erp_tutorial_hnn_core_v01.ipynb

```diff
@@ -148,9 +352,81 @@
     "## 3. Adding drives and running a simulation"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "5eecfe33",
+   "metadata": {},
+   "source": [
+    "First, we add another proximal drive at around 26 ms. The proximal drive represents a feedforward input from the lemniscal thalamus."
```

Comment:
changed the order of drives to be in increasing order of time rather than grouped by "proximal" and "distal" because it seems to follow the neuroscience story better that way